src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  let newKey;
+  let newValue;
+
+  for (const key in robot) {
+    newKey = robot[key];
+    newValue = key;
+
+    const keyArr = Object.keys(robot);
+
+    delete robot[key];
+
+    if (keyArr.includes(newKey)) {
+      return null;
+    }
+
+    robot[newKey] = newValue;
```

Comment:
No need to mutate initial value. Better to create new object and add values here. In such way your code will be more descriptive and simple: 
```
const result = {};
let newKey;
let newValue;

for (const key in robot) {
  ...
  result[newKey] = newValue;
}

return result;
```